test/scalability/mzcompose.py

```diff
@@ -42,6 +42,8 @@
     Postgres(),
 ]
 
+REGRESSION_THRESHOLD_IN_PERCENT = 0.2
```

Comment:
so 20? Or the name of the variable and the `max_deviation_in_percent` constructor argument are misleading?

Reply:
I vote for changing the variable name to something like `REGRESSION_THRESHOLD`

Reply:
I am not entirely convinced (see also https://stackoverflow.com/questions/48585623/percentage-stored-as-a-fraction-of-1-what-is-this-called and https://stackoverflow.com/questions/4840095/naming-convention-for-0-0-1-0-value-range). `AS_PERCENT_DECIMAL`?